library/src/main/java/com/doomonafireball/betterpickers/radialtimepicker/AmPmCirclesView.java

```diff
@@ -38,8 +39,7 @@
     private static final String TAG = "AmPmCirclesView";
 
     // Alpha level for selected circle.
-    private static final int SELECTED_ALPHA = Utils.SELECTED_ALPHA;
-    private static final int SELECTED_ALPHA_THEME_DARK = Utils.SELECTED_ALPHA_THEME_DARK;
+    private static final int SELECTED_ALPHA = 51;
```

Comment:
Why do you use a Magic value now instead of the one coming from `Utils` as previously? (this comment si valid for your two classes where this change has been made)